src/components/ViewPCOL/NotesForm.tsx

```diff
@@ -112,32 +108,31 @@ const NotesForm = ({
       <br />
 
       <div style={{ width: "100%", display: "flex" }}>
-        <Button
-          disabled={false /*addNote.isLoading*/}
-          onClick={() => closeDrawer()}
-        >
+        <Button disabled={addNote.isPending} onClick={() => closeDrawer()}>
           Cancel
         </Button>
         <Button
           appearance="primary"
           style={{ marginLeft: "auto" }}
-          // disabled={addNote.isLoading}
+          disabled={addNote.isPending}
           onClick={() => {
-            // addNote.mutateAsync(newNoteText).then(() => {
-            //   closeDrawer();
-            //   if (toSelections.length > 0) {
-            //     const email = {
-            //       To: [] as string[],
-            //       CC: [] as string[],
-            //       Subject: subject,
-            //       Body: body,
-            //     };
-            //     toSelections.forEach((person) => email.To.push(person.EMail));
-            //     ccSelections.forEach((person) => email.CC.push(person.EMail));
-            //     sendMail.mutate({ email, pcolId });
-            //   }
-            //   addNote.reset();
-            // });
+            addNote.mutateAsync(newNoteText).then(() => {
+              closeDrawer();
+              if (toSelections.length > 0) {
```

Comment:
Should probably add an OR condition of the CC field have a length. If they fill just the CC and not the To, than it doesn't warn them, and no email is sent.